app/javascript/app/document-capture/components/submission-pending.jsx

```diff
@@ -1,11 +1,18 @@
-import React from 'react';
+import React, { useRef, useEffect } from 'react';
 import Image from './image';
 
 function SubmissionPending() {
+  const headingRef = useRef(/** @type {?HTMLHeadingElement} */ (null));
+  useEffect(() => {
+    headingRef.current.focus();
+  }, []);
+
   return (
     <div>
       <Image assetPath="id-card.svg" alt="" width="216" height="116" />
-      <h2>We are processing your images…</h2>
+      <h2 ref={headingRef} tabIndex={-1}>
+        We are processing your images…
+      </h2>
       <p>This might take up to a minute. We’ll load the next step automatically when it’s done.</p>
       <p>Thanks for your patience!</p>
```

Comment:
I'm going to split out a separate pull request to translate these strings. I'll edit this comment when it's ready.